CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Unreleased
 
+## 1.18.0 / 2023-12-22
+
+* [FEATURE] promlint: Allow creation of custom metric validations. #1311
+* [FEATURE] api: Change HTTP Client into interface, so users can use customized clients, e.g. with retries on 429. #1387
+* [BUGFIX] histograms: Add timer to reset ASAP after bucket limiting has happened. #1367
+* [BUGFIX] testutil: Fix comparison of metrics with empty Help strings. #1378
+* [BUGFIX] Fix building client_golang in wasip1 OS. #1350
+* [ENHANCEMENT] Improved performance of `MetricVec.WithLabelValues(...)`. #1360
```

Comment:
Lol, nice great oneliner 💪🏽

Reply:
I literally copied the PR title :P

Reply:
I meant the change for optimization was only one line 🙈

Reply:
ah yes, great job by the grafana folks :)